docs/Antora/modules/background_info/pages/streaming.adoc

```diff
@@ -1,10 +1,67 @@
 = Streaming
 
-:note-caption: TODO
-[NOTE]
-====
-* Streaming types
-* Creating and using Streaming objects
-* How to manage Streaming sources per signal
-* Streaming connection heuristic usage
-====
\ No newline at end of file
+Streaming in the openDAQ(TM) architecture refers to the client side mechanism for continuous and real-time
+receiving of data from a data acquisition xref:glossary:glossary.adoc#device[Device], often facilitated by
+the use of protocols like xref:glossary:glossary.adoc#websocket_streaming_protocol[Websocket streaming], or
+xref:glossary:glossary.adoc#native_streaming_protocol[openDAQ(TM) native streaming]. The data transmission
+follows a subscription-based model, where the client subscribes to xref:glossary:glossary.adoc#signal[Signal]s'
+data produced by the Device. Streaming mechanism involves the receiving data in binary format over a
+communication channel (e.g. network), and deserialization it into openDAQ(TM)
+xref:glossary:glossary.adoc#data_packet[Packets]. The Packets are added to <<mirrored_signals,Mirrored Signal>>
+which may be connected to xref:glossary:glossary.adoc#input_port[Input Ports].
+
+The generalized Streaming mechanism serves to decouple data streaming from the Device structure publishing.
+
+== Streaming instantiation
+
+A specific Streaming protocol implementation is associated with an openDAQ(TM) xref:background_info:modules.adoc[Module].
+The SDK handles the routing and delegation of Streaming object instantiation to the appropriate module
+through using the connection string. The connection string always starts with the protocol type ID as its
+prefix, followed by a set of parameters unique to the protocol type. Some parameters, such as the IP address,
+might be obligatory, while others, like the port number, are optional and adopt default values if not
+explicitly specified.
+
+The openDAQ(TM) supports two types of xref:glossary:glossary.adoc#device[Device]s: those with a complete tree structure and those (also called
+a pseudo-devices) with a flat list of xref:glossary:glossary.adoc#signal[Signal]s. Regardless of the Device
```

Comment:
I'd add a brief explanation of the differences:

- Pseudo: no information of structure (eg. function blocks, subdevices, input ports), only flat list of signals. Created by streaming.
- Full device: Gets structure information from structure transfer modules, streaming hooks onto those signals and is only used for data transfer.

Reply:
Explanation added